lib/Db/ColumnTypes/UsergroupColumnQB.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace OCA\Tables\Db\ColumnTypes;
+
+use OCP\DB\QueryBuilder\IQueryBuilder;
+
+class UsergroupColumnQB extends SuperColumnQB implements IColumnTypeQB {
+	public function passSearchValue(IQueryBuilder $qb, string $unformattedSearchValue, string $operator, string $searchValuePlaceHolder): void {
+		// TODO how to handle searching for multiple users/groups?
```

Comment:
Do you mean, if multiple users/groups are part of the search string?

Reply:
Yeah. But I guess we can consider this later on

Reply:
Yes, I think this also needs more advanced capabilities of building view filters to not only be able to select a string value but also be able to differentiate between the different value types. Something to file an issue for later